src/aragonjs-wrapper.js

```diff
@@ -73,13 +73,20 @@ const prepareFrontendApps = (apps, daoAddress, gateway) => {
       // so the absolute path can be resolved from baseUrl.
       const startUrl = removeStartingSlash(app['start_url'] || '')
       const src = baseUrl ? resolvePathname(startUrl, baseUrl) : ''
+      const appName = app['appName']
```

Comment:
You could've: `const { appName } = app`

Reply:
Neat trick, thanks Gorka. Added it in https://github.com/aragon/aragon/pull/610/commits/eaa645e5a5faa9ed368dda66eee6b2c486540e52